Source/WebKit/WebProcess/WebCoreSupport/gtk/WebDragClientGtk.cpp

```diff
@@ -65,16 +64,39 @@ static RefPtr<ShareableBitmap> convertCairoSurfaceToShareableBitmap(cairo_surfac
 }
 #endif
 
+#if USE(SKIA)
+static RefPtr<ShareableBitmap> convertSkiaImageToShareableBitmap(SkImage* image)
+{
+    if (!image)
+        return nullptr;
+
+    IntSize imageSize(image->width(), image->height());
+    RefPtr bitmap = ShareableBitmap::create({ imageSize });
+    auto graphicsContext = bitmap->createGraphicsContext();
+
+    ASSERT(graphicsContext->hasPlatformContext());
+    auto* canvas = graphicsContext->platformContext();
+    canvas->drawImage(image, 0, 0);
```

Comment:
This could be just `graphicsContext->platformContext()->drawImage(image, 0, 0);`